static/app/components/events/interfaces/spans/spanTree.tsx

```diff
@@ -469,12 +469,10 @@ class SpanTree extends Component<PropType> {
         const {span, treeDepth, continuingTreeDepths} = payload;
 
         if (payload.type === 'span_group_chain') {
-          const groupingContainsAffectedSpan =
-            isEmbeddedSpanTree &&
```

Comment:
I'm not sure if there's a reason we excluded non-embedded trees apart from the fact it wasn't implemented